docs/modules/ROOT/pages/how-to-guides/proc_managing-compliance-with-the-enterprise-contract.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: PROCEDURE
+
+[id="managing-compliance-with-the-enterprise-contract_{context}"]
+= Managing compliance with the Enterprise Contract
+
+The Enterprise Contract (EC) is an artifact verifier and customizable policy checker. The Enterprise Contract helps you keep your software supply chain secure and ensures that the container images comply with your organization's policies. It does this by verifying the security and provenance of builds created through {ProductName}.
+
+{ProductName}'s build process utilizes the link:https://tekton.dev/docs/chains/[Tekton Chains] to generate a signed link:https://in-toto.io/in-toto/[in-toto] attestation of the build pipeline. The Enterprise Contract then uses this attestation to verify the build's integrity and compliance with a set of policies. These policies include best practices and any organization-specific requirements.
+
+.Prerequisites
+
+* You have created an application. 
+* You have an upgraded build pipeline.
+
+.Procedure
+
+. Open an existing application and go to the *Integration tests* tab.
+. Select *Add integration test*.
+. In the *Integration test name* field, enter a name of your choosing.
+. In the *Image bundle* field, enter *quay.io/redhat-appstudio-tekton-catalog/pipeline-enterprise-contract:devel.*
+
++
+TIP: You can find the source for the Tekton pipelines defined in the bundle in the link:https://github.com/redhat-appstudio/build-definitions/blob/main/pipelines/enterprise-contract.yaml[build-definitions] repository. If you want to use a specific version of the pipeline bundle instead of the devel tag, you can select one of the link:https://quay.io/repository/redhat-appstudio-tekton-catalog/pipeline-enterprise-contract?tab=tags[pinned tags]. Each tag name corresponds top a commit sha in the build-definitions repo. If you want be extra secure, you can also use a container image digest to pin it down.
```

Comment:
"extra secure" doesn't sound good. How about:
```suggestion
TIP: You can find the source for the Tekton pipelines defined in the bundle in the link:https://github.com/redhat-appstudio/build-definitions/blob/main/pipelines/enterprise-contract.yaml[build-definitions] repository. If you want to use a specific version of the pipeline bundle instead of the devel tag, you can select one of the link:https://quay.io/repository/redhat-appstudio-tekton-catalog/pipeline-enterprise-contract?tab=tags[pinned tags]. Each tag name corresponds top a commit sha in the build-definitions repo. For greater stability you could also use the container image digest in the image ref.
```